kiwi/boot/functions.sh

```diff
@@ -5517,6 +5517,21 @@ function killShell {
     fi
 }
 #======================================
+# getDeviceTransportType
+#--------------------------------------
+function getDeviceTransportType {
+    # /.../
+    # function to get the transport type of the
+    # given device. It uses lsblk command.
+    # ----
+    local device=$1
+    if [ -z "$device" ] || [ ! -e "$device" ];then
+        echo 1 ; return 1;
+    fi
+    local disk=${device%%[0-9]*}
+    echo "`lsblk -dno tran $disk`"
```

Comment:
@davidcassany Graveticks are deprecated and is limited and the `$()` is new. Whilst graveticks can be nested with a back-slash form, multi-nesting gets very bad and overall limited. The new `$()` form can be easily nested and can also contain a script. You can also put an arithmetic expansions, several sub-shells, complex relational expressions, like:
```bash
while [ $(( (($x + $y)/($a * $b)) < ($foo*$bar) )) -ne 0 ]
```
...and so on. Graveticks is an old thing that basically not used nowadays anymore.